iOSNativeLoginTemplate/iOSNativeLoginTemplate/NativeLoginViewFactory.swift

```diff
@@ -28,11 +28,35 @@
 import Foundation
 import SwiftUI
 
-@objc class NativeLoginViewFactory: NSObject {
-
-    @objc static func create() -> UIViewController {
-        let view = UIHostingController(rootView: NativeLoginView())
-
+class NativeLoginViewFactory: NSObject {
+    
+    static func create(
+        navigationPathObservable: NavigationPathObservable,
+        reCaptchaClientObservable: ReCaptchaClientObservable
+    ) -> UIViewController {
+        let view = NativeLoginTemplateHostingController(rootView: NativeLoginView()
+            .environmentObject(navigationPathObservable)
```

Comment:
I thought there might be a way to add environment objects to the app or scene in such a way `NativeLoginView` would automatically pick them up, but it wouldn't bind.  So, there're added explicitly here.  As the root/content view, these objects are then available all the way down the view hierarchy.

I'd be curious to review the app and scene configuration to see if this can be improved.